### PR TITLE
[KOGITO-9600][KSW-Guides]: Update chrome extension docs

### PR DESCRIPTION
**Pull request by AjayJagan**
**JIRA:https://issues.redhat.com/browse/KOGITO-9600**

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Review by paulovmr — APPROVED** [review]

**Comment by domhanak** [comment]
@tomasdavidorg  please  check

**Review by tomasdavidorg — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc [hunk 1]
```diff
@@ -53,16 +53,23 @@ For more information, see <<proc-install-chrome-extension-sw-editor, Installing
...
-. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using *See as source* button.
-. Make changes to the source code, add a commit message. 
 +
-You can either commit the changes to a branch directly or create a new branch for pull request.
+--
+.Serverless Workflow file in GitHub(edit mode)
+image::tooling/serverless-workflow-editor/swf-editor-in-github-edit.png[]
+--
++
+. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using *See as source* button.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:67]:
```suggestion
. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the *See as source* button.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc [hunk 2]
```diff
@@ -53,16 +53,23 @@ For more information, see <<proc-install-chrome-extension-sw-editor, Installing
...
+. Make changes to the source code, add a commit message. You can either commit the changes to a branch directly or create a new branch for pull request.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:68]:
```suggestion
. Make changes to the source code, and add a commit message. You can either commit the changes to a branch directly or create a new branch for pull requests.
```

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc [hunk 3]
```diff
@@ -53,16 +53,23 @@ For more information, see <<proc-install-chrome-extension-sw-editor, Installing
...
+. In a pull request you can switch between the original and the changes using the buttons provided.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:69]:
```suggestion
. In a pull request, you can switch between the original and the changes using the buttons provided.
```

**Review by kaldesai — APPROVED** [review]
@AjayJagan minor suggestions! Otherwise, your content looks good to me.

**Comment by ricardozanini** [comment]
Do we want this for 1.40 (TP1)?

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc [hunk 4]
```diff
@@ -53,16 +53,23 @@ For more information, see <<proc-install-chrome-extension-sw-editor, Installing
...
-. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using *See as source* button.
-. Make changes to the source code, add a commit message. 
 +
-You can either commit the changes to a branch directly or create a new branch for pull request.
+--
+.Serverless Workflow file in GitHub(edit mode)
+image::tooling/serverless-workflow-editor/swf-editor-in-github-edit.png[]
+--
++
+. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the *See as source* button.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:67]:
```suggestion
. You can switch to the full-screen mode. Also, you can hide the editor and view the source code using the `See as source` button.
```

**Review by radtriste — APPROVED** [review]
minor comment

**Comment by ricardozanini** [comment]
@paulovmr which versions we should backport this? Can we merge it?

**Review by MarianMacik — APPROVED** [review]

**Pull request merged**